src/main/kotlin/no/nav/tilleggsstonader/sak/behandling/fakta/BehandlingFaktaDto.kt

```diff
@@ -12,14 +12,28 @@ import no.nav.tilleggsstonader.sak.felles.domain.BarnId
 import java.time.LocalDate
 import java.time.LocalDateTime
 
-data class BehandlingFaktaDto(
-    val søknadMottattTidspunkt: LocalDateTime?,
-    val hovedytelse: FaktaHovedytelse,
-    val aktivitet: FaktaAktivtet,
-    val barn: List<FaktaBarn>,
-    val dokumentasjon: FaktaDokumentasjon?,
-    val arena: ArenaFakta?,
-)
+sealed class BehandlingFaktaDto(
```

Comment:
Tror det kan være en fordel å sende med en type til frontend for å enklere håndtere dette? 
Se `VedtakTilsynBarnDto`
```kotlin
@JsonTypeInfo(use = JsonTypeInfo.Id.NAME)
@JsonSubTypes(
    JsonSubTypes.Type(InnvilgelseTilsynBarnDto::class, name = "INNVILGELSE"),
    JsonSubTypes.Type(AvslagTilsynBarnDto::class, name = "AVSLAG"),
)
```

Reply:
Hvis man har med denne så trenger ikke frontend å typcasta, men kan sjekke `@type == Læremidler` så vet den hvilken type det er

Reply:
Der må man typecasta, som man ikke trenger hvis man sjekker typen

Reply:
Fixed: 8a8a3b707377f12cccc09a090c500611a15bbb44